Buy/Global.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Global.swift
+//  Buy
+//
+//  Created by Dima Bart on 2017-02-24.
+//  Copyright © 2017 Shopify Inc. All rights reserved.
+//
+
+import Foundation
+
+private final class BundleCoordinator: NSObject {
+    
+    static func frameworkBundle() -> Bundle {
+        return Bundle(for: self.classForCoder())
+    }
+}
+
+struct Global {
```

Comment:
Honestly, it's a terrible name and I'd love a better suggestion. The intention was to provide access globally available information like bundle identifiers.